employees/tests/test_unit_custom_mixins.py

```diff
@@ -126,9 +126,11 @@ def test_month_navigator_get_context_data_method_should_return_all_data_necessar
             "path": self.mixin.request.path,
             "navigation_text": MonthNavigationText,
             "month_form": MonthSwitchForm(initial_date=datetime.date(year=2019, month=3, day=1)),
-            "next_month_url": "/reports/project/1/2019/4/",
-            "recent_month_url": f"/reports/project/1/{current_date.year}/{current_date.month}/",
-            "previous_month_url": "/reports/project/1/2019/2/",
+            "next_month_url": reverse("project-report-list", args=(self.mixin.kwargs["pk"], 2019, 4)),
```

Comment:
I see that sometimes we use `args` and sometimes `kwargs` in url building. I would suggest that we don't mix them up. My personal choice would be `kwargs` (greater readability). @rwrzesien, what do you think?

Reply:
I would also prefer `kwargs` because we can see all keys and values associated with them. If you agree @rwrzesien I can change it for you and merge today :)

Reply:
Yes, I prefer `kwargs` too, I just do it in little rush yesterday, go ahead and fix it if you want :)

Reply:
I have updated it.